#DPATCHLEVEL=1

2004-09-15  Guillem Jover  <guillem@hadrons.org>

	* linux/dev/include/linux/skbuff.h (skb_put, skb_push): Fix errors
	for deprecated use of labels at end of compound statements.
	* i386/i386/fpe_linkage.c: Forward declare enable_fpe.


diff -Naur gnumach-20040229.orig/linux/dev/include/linux/skbuff.h gnumach-20040229/linux/dev/include/linux/skbuff.h
--- gnumach-20040229.orig/linux/dev/include/linux/skbuff.h	26 Apr 1999 05:48:00 -0000	1.1
+++ gnumach-20040229/linux/dev/include/linux/skbuff.h	15 Sep 2004 12:52:34 -0000
@@ -405,8 +405,8 @@
 	if(skb->tail>skb->end)
 	{
 		__label__ here;
-		panic("skput:over: %p:%d", &&here,len);
 here:
+		panic("skput:over: %p:%d", &&here,len);
 	}
 	return tmp;
 }
@@ -418,8 +418,8 @@
 	if(skb->data<skb->head)
 	{
 		__label__ here;
-		panic("skpush:under: %p:%d", &&here,len);
 here:
+		panic("skpush:under: %p:%d", &&here,len);
 	}
 	return skb->data;
 }
diff -Naur gnumach-20040229.orig/i386/i386/fpe_linkage.c gnumach-20040229/i386/i386/fpe_linkage.c
--- gnumach-20040229.orig/i386/i386/fpe_linkage.c	2004-09-15 16:10:15.000000000 +0200
+++ gnumach-20040229/i386/i386/fpe_linkage.c	2004-09-15 16:09:23.000000000 +0200
@@ -84,6 +84,7 @@
 	((struct real_gate *)&curr_idt(mycpu)[idx])
 
 void	set_user_access();	/* forward */
+void	enable_fpe(register struct i386_fpsave_state *ifps);
 
 /*
  * long pointer for calling FPE register recovery routine.
